action_server/src/action_server/actions/action.py

```diff
@@ -107,16 +107,19 @@ def _configure(self, robot, config):
         raise NotImplementedError
 
     def start(self):
-        rospy.loginfo("Starting executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Starting executing of action {}.".format(self.get_name()))
         self._start()
         return self._execute_result
 
     def _start(self):
         raise NotImplementedError
 
     def cancel(self):
-        rospy.loginfo("Canceling executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Canceling executing of action {}.".format(self.get_name()))
         return self._cancel()
 
     def _cancel(self):
         raise NotImplementedError
+
+    def get_name(self):
+        return self.__class__.__name__
```

Comment:
Is it possible to multiple instances of the same Action-subclass? If so, only the name isn't enough to distinguish between instances when debugging

Reply:
Yes, that is possible. But this is not for debugging. How do you suggest I change this? I think the function name does not necessarily imply a unique name...

Reply:
If only for external info/feedback purposes, then ignore my comment.